.github/workflows/test.yml

```diff
@@ -66,6 +66,9 @@ jobs:
             os: ubuntu-latest
           - target: x86_64-unknown-linux-musl
             os: ubuntu-latest
+          - target: x86_64-unknown-linux-musl
+            os: ubuntu-latest
+            flags: --no-default-features
```

Comment:
As far as I can see in the checks of the corresponding commit this doesn't actually add another musl build, but instead overwrites the previously defined one: You can see that there's still only one musl build, but it now uses --no-default-features.

I guess `include` doesn't let you add new builds to the matrix, but only parameters to existing matrix items, and the matrix's set up based on `target` and `rust` only.

I very much want to retain the musl build with default feature since that's the release binary for Github pages, so please do change this and make sure that all current builds, and above all, those for release binaries remain intact.

Can't we cross-build for e.g. `powerpc64-unknown-linux-gnu`?     After all, as far as I understood our discussion, that's the target you're actually interested in, isn't it?

Reply:
Indeed, `powerpc{,64,64le}`. Good idea, thanks.

Reply:
Don't know whether that's easy though… don't have much experience in cross-building on Github actions :slightly_smiling_face: